ceph_deploy/mon.py

```diff
@@ -232,11 +232,9 @@ def mon_add(args):
 
 
 def mon_create(args):
-
     cfg = conf.ceph.load(args)
     if not args.mon:
-        mon_initial_members = cfg.safe_get('global', 'mon_initial_members')
-        args.mon = re.split(r'[,\s]+', mon_initial_members)
+        args.mon = get_mon_initial_members(args)
 
```

Comment:
I see what you are talking about.

Here's what I would do...

Add _cfg=None to the method signature get_mon_initial_members().  That way you can pass in a load cfg if you already have one.  if not, make it load one.  I'll populate the rest of the comments to that effect.